bogenliga/bogenliga-db-migration/src/main/resources/db/migration/all/V51__tablet_session__create.sql

```diff
@@ -9,6 +9,7 @@ CREATE TABLE tablet_session (
     tablet_session_match_id       DECIMAL(19, 0),
     tablet_session_satznr         DECIMAL(1, 0) DEFAULT 1,
     is_active                     BOOLEAN NOT NULL DEFAULT TRUE,
+    access_token                  DECIMAL(19,0) DEFAULT 0,
```

Comment:
Das wird nicht funktionieren! Flyway meckert dann, dass das Skript geändert wurde.
Auslagern in ein neues Skript (vgl. beim letzten Mal Benutzer Update!) @TS-Tobias weiß Bescheid.
Das neue Skript sollte V60__alter__table__tablet__session oder sowas heißen (zwei Unterstriche) 
Außerdem die Skripte auch in bogenliga-application/src/main/resources/db/migration/all ablegen

Reply:
habe ich entfernt und in V59 umgezogen